tests-legacy/Unit/Classes/ValidateCoreTest.php

```diff
@@ -365,4 +373,24 @@ public function isFloatDataProvider()
             )
         );
     }
+
+    public function isArrayWithIdsDataProvider()
+    {
+        return array(
+            array(false, 'This is not an array'),
```

Comment:
```suggestion
            array(false, 42),
            array(false, '42'),
```
Then these ones as well I guess?